app/scenes/Document/components/Header.tsx

```diff
@@ -314,19 +315,23 @@ function DocumentHeader({
             )}
             {can.update && isDraft && !isRevision && (
               <Action>
-                <Tooltip
-                  tooltip={t("Publish")}
-                  shortcut={`${metaDisplay}+shift+p`}
-                  delay={500}
-                  placement="bottom"
-                >
-                  <Button
-                    onClick={handlePublish}
-                    disabled={publishingIsDisabled}
+                {document.collectionId ? (
```

Comment:
Replace this check with `false` to test out the flow in dev. This is so because we don't yet have the capability(via UI) to create a doc without collection.

Reply:
Not needed anymore. Added the capability to create a doc without collection.